safe-global/web-core
# fix: display SignOrExecuteForm on opening Change threshold modal

## What it solves

Resolves #751

## How this PR fixes it
The `<ChangeThresholdDialog />` renders `<SignOrExecuteForm />` unconditionally and passes `undefined` safeTx to disable the "Submit" button.

## How to test it
1. Open the Safe settings and click to change threshold.
2. Submit button should be visible and disabled until the selected threshold differs from the current one.

## Analytics changes
None

## Screenshots
<img width="634" alt="Screenshot 2022-10-04 at 15 55 21" src="https://user-images.githubusercontent.com/32431609/193838431-5d41847f-25b5-4684-89f1-0888bd20e9c2.png">
